[2025-03-26T00:52:39.296+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: titanic_survival_classifier.model_validation_task manual__2025-03-26T00:26:14.173301+00:00 [queued]>
[2025-03-26T00:52:39.315+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: titanic_survival_classifier.model_validation_task manual__2025-03-26T00:26:14.173301+00:00 [queued]>
[2025-03-26T00:52:39.316+0000] {taskinstance.py:2170} INFO - Starting attempt 3 of 3
[2025-03-26T00:52:39.342+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): model_validation_task> on 2025-03-26 00:26:14.173301+00:00
[2025-03-26T00:52:39.355+0000] {standard_task_runner.py:60} INFO - Started process 1865 to run task
[2025-03-26T00:52:39.361+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'titanic_survival_classifier', 'model_validation_task', 'manual__2025-03-26T00:26:14.173301+00:00', '--job-id', '21', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpqkh8ammn']
[2025-03-26T00:52:39.363+0000] {standard_task_runner.py:88} INFO - Job 21: Subtask model_validation_task
[2025-03-26T00:52:39.389+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-03-26T00:52:39.455+0000] {task_command.py:423} INFO - Running <TaskInstance: titanic_survival_classifier.model_validation_task manual__2025-03-26T00:26:14.173301+00:00 [running]> on host 0f3b2e2ddf0a
[2025-03-26T00:52:39.584+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='titanic_survival_classifier' AIRFLOW_CTX_TASK_ID='model_validation_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-26T00:26:14.173301+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-26T00:26:14.173301+00:00'
[2025-03-26T00:52:39.663+0000] {logging_mixin.py:188} INFO - ---Testing Model---
[2025-03-26T00:52:39.687+0000] {logging_mixin.py:188} INFO - Test predictions complete
[2025-03-26T00:52:39.729+0000] {logging_mixin.py:188} INFO - Submission saved at /opt/***/data/submission/submission.csv
[2025-03-26T00:52:39.730+0000] {python.py:201} INFO - Done. Returned value was: {'submission_file': '/opt/***/data/submission/submission.csv', 'test_predictions': [0, 0, 0, 0, 1, 0, 1, 0, 1, 0, 0, 0, 1, 0, 1, 1, 0, 0, 1, 0, 0, 0, 1, 1, 1, 0, 1, 0, 0, 0, 0, 0, 0, 0, 1, 0, 1, 1, 0, 0, 0, 1, 0, 1, 1, 0, 0, 0, 1, 0, 1, 0, 1, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 1, 1, 1, 1, 0, 0, 1, 1, 0, 1, 1, 1, 1, 0, 1, 0, 1, 0, 0, 0, 0, 0, 0, 1, 1, 1, 0, 1, 0, 1, 0, 0, 0, 1, 0, 1, 0, 1, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 1, 1, 1, 1, 0, 0, 1, 0, 1, 1, 0, 1, 0, 0, 1, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0, 0, 0, 0, 1, 1, 0, 1, 1, 0, 1, 0, 0, 1, 0, 0, 1, 1, 0, 0, 0, 0, 0, 1, 1, 0, 1, 1, 0, 1, 1, 0, 1, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 0, 1, 1, 0, 1, 1, 0, 0, 1, 0, 1, 0, 0, 0, 0, 1, 0, 0, 1, 0, 1, 0, 1, 0, 1, 0, 1, 1, 0, 1, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 1, 0, 0, 1, 1, 0, 1, 1, 1, 0, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 1, 1, 0, 0, 0, 0, 1, 0, 0, 0, 1, 1, 0, 1, 0, 0, 0, 0, 1, 0, 1, 1, 1, 0, 0, 1, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 1, 0, 0, 0, 1, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 1, 0, 1, 0, 1, 1, 0, 0, 0, 1, 0, 1, 0, 0, 1, 0, 1, 1, 0, 1, 0, 0, 1, 1, 0, 0, 1, 0, 0, 1, 1, 1, 0, 0, 0, 0, 0, 1, 1, 0, 1, 0, 0, 0, 0, 1, 1, 0, 0, 0, 1, 0, 1, 0, 0, 1, 0, 1, 0, 1, 1, 0, 0, 1, 1, 1, 1, 1, 0, 1, 0, 0, 0]}
[2025-03-26T00:52:39.774+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=titanic_survival_classifier, task_id=model_validation_task, execution_date=20250326T002614, start_date=20250326T005239, end_date=20250326T005239
[2025-03-26T00:52:39.854+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-03-26T00:52:39.890+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
